ReSwiftTests/StoreSubscriberTests.swift

```diff
@@ -116,8 +141,9 @@ class StoreSubscriberTests: XCTestCase {
         let subscriber = TestFilteredSubscriber<TestCustomAppState.TestCustomSubstate>()
 
         store.subscribe(subscriber) {
-            $0.select { $0.substate }
-                .skipRepeats { $0.value == $1.value }
+            $0
```

Comment:
same here: rather introduce a `_RegularSubstateSelection_` and `_WithKeyPaths_` in the test names to disambiguate the duplicates than remove the old ones